project/GenFactories.scala

```diff
@@ -533,7 +532,7 @@ $endif$
    *                 ^
    * </pre>
    */
-  def and(containWord: ContainWord): AndContainWord = new AndContainWord
+  def and(containWord: ContainWord)(implicit prettifier: Prettifier, pos: source.Position): AndContainWord = new AndContainWord(prettifier, pos)
```

Comment:
Why do and and or need the implicit prettifier and position here? Why can't the one captured by should be passed down?